package.json

```diff
@@ -84,7 +87,7 @@
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.33.0",
     "chai": "^4.1.2",
-    "eslint": "^7.0.0",
+    "eslint": "^5.16.0",
```

Comment:
Why use a minor version? From 7 to 5.16

Reply:
Oops! I've updated everything now, It should be all the latest versions.